### PR TITLE
Added button and text view to login layout for register and forgot pa…

### PR DESCRIPTION
**Pull request by OnlyAlf**
…ssword. Added some new dimensions and strings to res files.

### CODE REVIEW

File: app/src/main/res/layout/activity_login.xml [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:paddingStart="@dimen/activity_horizontal_padding"
+    android:paddingBottom="@dimen/activity_vertical_padding"
     android:paddingEnd="@dimen/activity_vertical_padding"
-    android:paddingTop="@dimen/activity_horizontal_padding"
-    android:paddingBottom="@dimen/activity_vertical_padding">
+    android:paddingStart="@dimen/activity_horizontal_padding"
+    android:paddingTop="@dimen/activity_horizontal_padding">
```
Comment (carlosjuarez) [inline app/src/main/res/layout/activity_login.xml]: check the places of the changes you are submitting, 

remember what i told you about the merge "to let all the things from others in his place, and to add yours at the end"

File: app/src/main/res/layout/activity_login.xml [hunk 2]
```diff
@@ -37,7 +36,7 @@
         android:layout_marginBottom="@dimen/margin_top_bottom"
         android:layout_marginEnd="@dimen/input_margin_start_to_end"
         android:layout_marginStart="@dimen/input_margin_start_to_end"
-        android:layout_marginTop="@dimen/input_margin_start_to_end"
+        android:layout_marginTop="@dimen/margin_top_bottom"
```
Comment (carlosjuarez) [inline app/src/main/res/layout/activity_login.xml:40]: same here, you are deleting the input_margin_start_to_end dimension in this line, why?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -20,7 +22,6 @@
     <string name="feeditem_descriptiondefault">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed suscipit sodales dolor in sodales. Duis</string>
     <string name="feed_item_location_default">Street Name, Num, City, State, ZipCode</string>
     <string name="feed_item_price_default">$ $ $ $</string>
-
```
Comment (carlosjuarez) [inline app/src/main/res/values/strings.xml]: you are deleting this line of code

**Review by carlosjuarez — CHANGES_REQUESTED** [review]
Check the comments and update your commit

File: app/src/main/res/layout/activity_login.xml [hunk 4]
```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:paddingBottom="@dimen/activity_vertical_padding"
     android:paddingEnd="@dimen/activity_vertical_padding"
-    android:paddingStart="@dimen/activity_horizontal_padding"
-    android:paddingTop="@dimen/activity_horizontal_padding">
```
Comment (carlosjuarez) [inline app/src/main/res/layout/activity_login.xml]: this are in the wrong place

**Review by ecejmsr — APPROVED** [review]
Looks good

**Pull request merged**